### PR TITLE
Post SR2019 Kit Collation

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
I'd very much like to get the booking confirmed before sending this out.

**Comment by trickeydan** [comment]
We should know about the booking tomorrow. I cc'd you when I sent the request, but it has to be handled by a human.

File: SR2019/2019-04-29-kit-collation.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+Hi,
+
+This coming Saturday, the 4th May, we'll be having a kit collation event to go
+through this years team kits, clean up and test the parts and collate them into
+boxes by part. This is similar to the activity weekends before this year's
+Kickstart, without the packaging of the parts into team kit boxes.
+
+We're still confirming our room booking at the University of Southampton but
```
Comment (PeterJCLaw) [inline SR2019/2019-04-29-kit-collation.md]: Need to work out how the people who turn up will actually know where to go.
Options include:
- a phone number (maybe the SRO number, maybe the competition phone)
- we'll post in Slack/they can post in Slack to be let in

Comment (RealOrangeOne) [reply SR2019/2019-04-29-kit-collation.md]: We also need to tell people _when_ to turn up. Namely when people will be around, and when the RUBs will actually turn up

File: SR2019/2019-04-29-kit-collation.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+to: All Volunteers
+subject: Collating team kits
+preview-text: We need your help to tidy away the kits from this year's competition
+---
+
+Hi,
+
+This coming Saturday, the 4th May, we'll be having a kit collation event to go
```
Comment (RealOrangeOne) [inline SR2019/2019-04-29-kit-collation.md]: Can we specify a time as well. At least an expected starting time, so people know when it's useful to turn up. An expected end time may also be useful, even if it's a guess we don't stick to.

Comment (PeterJCLaw) [reply SR2019/2019-04-29-kit-collation.md]: 73c8201

File: SR2019/2019-04-29-kit-collation.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2019/2019-04-29-kit-collation.md]: 73c8201

File: SR2019/2019-04-29-kit-collation.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+through this years team kits, clean up and test the parts and collate them into
+boxes by part. This is similar to the activity weekends before this year's
+Kickstart, without the packaging of the parts into team kit boxes.
+
+We'll be meeting in the Mountbatten building at the University of Southampton.
```
Comment (trickeydan) [inline SR2019/2019-04-29-kit-collation.md]: Should we add a link for travel directions? Those unfamilar with the university might not be able to find it

Comment (RealOrangeOne) [reply SR2019/2019-04-29-kit-collation.md]: In my experience, It's actually really easy to find your way to highfield, and around it. 2 minutes of googling is all it takes.

**Comment by RealOrangeOne** [comment]
This email has now shipped.

**Comment by trickeydan** [comment]
> This email has now shipped.

Why was this shipped when there were outstanding review comments?

**Pull request merged**